### PR TITLE
created file for links to c++ material

### PR DESCRIPTION
**Pull request by warhead38**

### CODE REVIEW

**Comment by pr4shan7** [comment]
Please describe your PR. @warhead38

**Comment by warhead38** [comment]
i have created a file in learning resources folder which contains study material links and suggestions for c++

File: learning-resources/c++ Links [hunk 1]
```diff
@@ -0,0 +1,12 @@
+(VIDEOS)
+
+(1) https://www.youtube.com/watch?v=vLnPwxZdW4Y
+(2) https://www.youtube.com/watch?v=MhYECGUzdA4
```
Comment (pr4shan7) [inline learning-resources/c++ Links]: Please do not use bare URLs, instead use the standard markdown format, for example, [name of video, from channel-name](link)

File: learning-resources/c++ Links [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+(SITES)
+
+(1) https://www.codecademy.com/learn/learn-c-plus-plus?utm_source=xinquji
```
Comment (pr4shan7) [inline learning-resources/c++ Links]: Same goes for links to websites.

**Pull request closed without merging**